### PR TITLE
Update CLI install commands

### PR DESCRIPTION
**Pull request by jwood803**
## Summary

Update ML.NET CLI install commands to latest.

[Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-ml-net-cli?branch=pr-en-us-27859)

### CODE REVIEW

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ The ML.NET CLI generates good quality ML.NET models and source code using automa
 
 ## Pre-requisites
 
-- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet/3.1)
+- [.NET 5](https://dotnet.microsoft.com/download/dotnet/5.0)
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:20]:
```suggestion
- [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
```

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 2]
```diff
@@ -29,10 +29,24 @@ Note: If after installing .NET SDK the `dotnet tool` command is not working, sig
...
 The ML.NET CLI is installed like any other dotnet Global Tool. You use the `dotnet tool install` .NET CLI command.
 
-The following example shows how to install the ML.NET CLI in the default NuGet feed location:
+The ML.NET CLI is available for Windows, macOS, and Linux.
 
-```dotnetcli
-dotnet tool install -g mlnet
+# [Windows](#tab/windows)
+
+```console
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:36]:
~~~suggestion
```dotnetcli
~~~

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 3]
```diff
@@ -29,10 +29,24 @@ Note: If after installing .NET SDK the `dotnet tool` command is not working, sig
...
+dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.Win64
+```
+
+# [Mac](#tab/mac)
+
+```console
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:42]:
~~~suggestion
```dotnetcli
~~~

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 4]
```diff
@@ -29,10 +29,24 @@ Note: If after installing .NET SDK the `dotnet tool` command is not working, sig
...
+dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.OSX
+```
+
+# [Linux](#tab/linux)
+
+```console
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:48]:
~~~suggestion
```dotnetcli
~~~

**Comment by luisquintanilla** [comment]
@jwood803 

The sections after the install command all appear under the Linux tab. 

image

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 5]
```diff
@@ -29,10 +29,24 @@ Note: If after installing .NET SDK the `dotnet tool` command is not working, sig
...
+dotnet tool install --global Microsoft.ML.ModelBuilder.CLI.Linux64
 ```
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:51]:
```suggestion

---

```

**Review by luisquintanilla — COMMENTED** [review]
@jwood803 can you please confirm what the update command is? For some reason using `mlnet` to update failed for me, but when I used the package name with the RID for my OS, it seemed to work.

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 6]
```diff
@@ -17,24 +17,67 @@ The ML.NET CLI generates good quality ML.NET models and source code using automa
 
 ## Pre-requisites
 
-- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet/3.1)
+- [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:20]:
```suggestion
- [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
```

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 7]
```diff
@@ -17,24 +17,67 @@ The ML.NET CLI generates good quality ML.NET models and source code using automa
...
 
-The following example shows how to install the ML.NET CLI in the default NuGet feed location:
+The ML.NET CLI is available for Windows, macOS, and Linux. There are also different versions of the CLI depending if you have Intel based processor or an ARM based processor.
+
+> [!NOTE]
+> Installations with the ARM based versions of the ML.NET CLI won't have access to the image classification task due to limitations on the processor.
+
+# [Windows](#tab/windows)
+
+## Intel
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:39]: I think we can update Intel with x64.
```suggestion
## x64
```

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 8]
```diff
@@ -17,24 +17,67 @@ The ML.NET CLI generates good quality ML.NET models and source code using automa
...
 
 Note: If after installing .NET SDK the `dotnet tool` command is not working, sign out from Windows and sign in again.
 
 ## Install
 
-The ML.NET CLI is installed like any other dotnet Global Tool. You use the `dotnet tool install` .NET CLI command.
+The ML.NET CLI is installed like any other dotnet Global Tool. You use the `dotnet tool install --global` .NET CLI command.
 
-The following example shows how to install the ML.NET CLI in the default NuGet feed location:
+The ML.NET CLI is available for Windows, macOS, and Linux. There are also different versions of the CLI depending if you have Intel based processor or an ARM based processor.
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:32]:
```suggestion
The ML.NET CLI is available for Windows, macOS, and Linux. There are also different versions of the CLI for x64 and ARM based processors.
```

**Review by luisquintanilla — COMMENTED** [review]
@jwood803 Looks good to me. I added a few comments. Let's hold off on publishing until dotnet/machinelearning-modelbuilder#2010 is addressed.

**Comment by jwood803** [comment]
@luisquintanilla I think this is now ready for another review for merging 😄

**Comment by luisquintanilla** [comment]
Closing PR. Changes made as part of #28338

**Pull request closed without merging**